src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonNode.To.cs

```diff
@@ -5,22 +5,14 @@ namespace System.Text.Json.Nodes
 {
     public abstract partial class JsonNode
     {
-        // trimming-safe default JsonSerializerOptions instance used by JsonNode methods.
-        private protected static readonly JsonSerializerOptions s_defaultOptions = new();
```

Comment:
No longer necessary since the `JsonNode.WriteTo` implementations no longer compose via intermediate `JsonConverter.Write` calls that require a `JsonSerializerOptions`.